src/main/java/util/DataConverterImpl.java

```diff
@@ -0,0 +1,26 @@
+package util;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> lines) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : lines) {
+            String[] parts = line.split(",");
```

Comment:
There is a potential risk of encountering an `ArrayIndexOutOfBoundsException` if the input `line` does not contain the expected number of parts after splitting. It's a good practice to validate the length of the `parts` array before accessing its elements.